workflow/react-project/src/components/common/Button/index.jsx

```diff
@@ -2,8 +2,8 @@ import clsx from 'clsx';
 
 import './style.less';
 
-const Button = ({ children, className, onClick }) => (
-  <button type="button" onClick={onClick} className={clsx('button', className)}>
+const Button = ({ children, className, onClick, type }) => (
+  <button type={type || 'button'} onClick={onClick} className={clsx('button', className)}>
```

Comment:
`type={type || 'button'} `- it's okay 
another way to set default value for props - `const Button = ({ children, className, onClick, type = 'button' })` - here if type will be `undefined` it'll set as `'button'`